closure-typing/storage.js

```diff
@@ -1,5 +1,24 @@
 function storage() {
+    let obj = {};
 
+    const store = function(...args) {
+        if (args.length === 1) {
```

Comment:
its great that you used spread operator here, but `arguments` would be enough for your task, again keeping thing simple is better way